src/status_im/accounts/login/core.cljs

```diff
@@ -292,13 +293,22 @@
                 (get-user-password address)))))
 
 (fx/defn open-login-callback
-  [{:keys [db] :as cofx} password]
-  (if password
+  [{:keys [db] :as cofx} password {:keys [bioauth-success bioauth-notrequired bioauth-message]}]
+  (if (and password
+           (or bioauth-success bioauth-notrequired))
     (fx/merge cofx
               {:db (assoc-in db [:accounts/login :password] password)}
               (navigation/navigate-to-cofx :progress nil)
               (user-login false))
-    (navigation/navigate-to-cofx cofx :login nil)))
+    (fx/merge cofx
+              (when bioauth-message
+                {:utils/show-popup {:title "biometric auth" :content bioauth-message}})
```

Comment:
this needs to be localized

Reply:
yeah, i'll do a localization update once we have a finalized version